docs/source/chaincode4ade.rst

```diff
@@ -39,18 +44,13 @@ In each language, the ``Invoke`` method is called by clients to submit transacti
 proposals. This method allows you to use the chaincode to read and write data on
 the channel ledger.
 
-You also need to include an ``Init`` method that will serve as the initialization
-function for your chaincode. This method will be called in order to initialize
-the chaincode when it is started or upgraded. By default, this function is never
-executed. However, you can use the chaincode definition to request that the ``Init``
-function be executed. If execution of ``Init`` is requested, fabric will ensure
-that ``Init`` is invoked before any other function and is only invoked once.
-This option provides you additional control over which users can initialize the
-chaincode and the ability to add initial data to the ledger. If you are using
-the peer CLI to approve the chaincode definition, use the ``--init-required``
-flag to request the execution of the ``Init`` function. Then call the ``Init``
-function by using the `peer chaincode invoke` command and passing the
-``--isInit`` flag. For more information, see :doc:`chaincode_lifecycle`.
+You also need to include an ``Init`` method in your chaincode that will serve as
+the initialization function. This function is required by the chaincode interface,
+but does not necessarily need to invoked by your applications. You can use the
+Fabric chaincode lifecycle process to specify whether the ``Init`` function must
+be called prior to Invokes. For more information, refer to the initialization
+parameter in the `Approving a chaincode definition <https://hyperledger-fabric.readthedocs.io/en/{BRANCH}/chaincode_lifecycle.html#step-three-approve-a-chaincode-definition-for-your-organization>`__
```

Comment:
Unfortunately relative links are not supported in rst (or at least not common that I could find them easily. This is a work around, but will only work until after the PR is merged (so the build will still have the old doc)

Reply:
[It looks like this might work](https://stackoverflow.com/questions/3630386/insert-relative-link-in-restructuredtext)?

Reply:
Sorry, I meant a relative link that goes to a header. But it looks like you can do that too! (I forgot somehow. I updated the PR with that.